ocaml/typing/typecore.ml

```diff
@@ -1489,14 +1490,16 @@ let type_comprehension_for_range_iterator_index ~loc ~env ~param tps =
        for duplicates or anything else. *)
     ~any:Fun.id
     ~var:(fun ~name ~pv_mode ~pv_type ~pv_loc ~pv_as_var ~pv_attributes ->
+      (* CR tnowak: verify this change *)
```

Comment:
This is fine for this PR. If we add `Uid.t`s to the bindings in for loops and comprehension bindings then they will need to come from here.

Reply:
I passed the Uid a bit further down and added comments, is that correct? 982ab55fbed40dcb062b5dc6731ed940b5deac29